hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/missingContainers/missingContainers.tsx

```diff
@@ -192,16 +519,41 @@ export class MissingContainers extends React.Component<Record<string, object>, I
     this.setState({
       loading: true
     });
-    axios.get('/api/v1/containers/missing').then(response => {
-      const missingContainersResponse: IMissingContainersResponse = response.data;
-      const totalCount = missingContainersResponse.totalCount;
-      const missingContainers: IMissingContainerResponse[] = missingContainersResponse.containers;
+
+    axios.all([
+      axios.get('/api/v1/containers/missing'),
+      axios.get('/api/v1/containers/unhealthy/UNDER_REPLICATED'),
+      axios.get('/api/v1/containers/unhealthy/OVER_REPLICATED'),
+      axios.get('/api/v1/containers/unhealthy/MIS_REPLICATED'),
+      axios.get('/api/v1/containers/unhealthy/ALL_REPLICAS_UNHEALTHY'),
+    ]).then(axios.spread((missingContainersResponse, underReplicatedResponse, overReplicatedResponse, misReplicatedResponse, allReplicatedResponse) => {
+      
+      const missingContainersResponse1: IMissingContainersResponse = missingContainersResponse.data;
```

Comment:
Why do we need to number (eg. `missingContainersResponse1`) the variables like this? I don't see other usages later.

Reply:
const missingContainersResponse1: IMissingContainersResponse = missingContainersResponse.data;
const totalCount = missingContainersResponse1.totalCount;
const missingContainers: IMissingContainerResponse[] = missingContainersResponse1.containers;

We are assigning to missingContainersResponse1.containers to missingContainers

Reply:
Changes Completed.